frontend/src/components/Layouts/HomePageComponent/HomePageComponent.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import CategoryOutput from './components/CategoryOutput';
+import Wrrapper from '../../Wrraper/Wrraper';
+import Carousel from './components/Carousel';
+import carouselData from '../../../data/carouselData.json';
+import styles from './HomePageComponent.module.scss';
+
+const HomePageComponent = () => {
+  return (
+    <Wrrapper>
+      <Carousel data={carouselData.slides} />
+      <h2 className={styles.title}>
+        <span>Акційна</span> ціна
+      </h2>
+      <CategoryOutput api={'v1/products/discounted?page=1&size=8'} link={'#'} />
```

Comment:
String is better to pass just as a string `api='v1/products/discounted?page=1&size=8'`